static/css/resources.css

```diff
@@ -0,0 +1,18 @@
+.content-header {
```

Comment:
I don't think you meant to commit this file (you have an identical copy in `extras/`).